### PR TITLE
[OSPP] The implementation for Observability management

### PR DESCRIPTION
**Pull request by Kanakami**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
users can only check the status, parameters and metrics of tasks via the command line after creating edge-cloud synergy AI tasks by sedna. This design is intended to provide observability management for displaying logs and metrics of sedna in real time.

This is the implementation of pr #340

### CODE REVIEW

File: docs/proposals/observability-management.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+* [Observability Management](#observability-management)
```
Comment (jaypume) [inline docs/proposals/observability-management.md:1]: Proposal is already in another PR, please remove this part in this PR.

File: exporter/hardSamplesExporter.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from prometheus_client import start_http_server,Gauge
```
Comment (jaypume) [inline exporter/hardSamplesExporter.py:1]: It is suggestted to split this script to two new example, and put them into "examples" directory. One is for JointInference, One is for IncrementalLearning. Then put them seperately into directory
`example/monitor/incremental_learning` and `example/monitor/joint_inference` .

File: deployment/grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,573 @@
+# Grafana Helm Chart
```
Comment (jaypume) [inline deployment/grafana/README.md:1]: It is found that this page is almost the same with [this page](https://artifacthub.io/packages/helm/grafana/grafana/6.29.2).
If it is the same, please just give a link somewhere, no need to copy all the content to here.

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Quite a few code is copied from https://github.com/grafana/helm-charts/blob/main/charts. KubeEdge authors shall not upload codes from other community as our own. 

Suggestion: 
1. write a markdown to ask others to download grafana and how to adjust yaml files to fit sedna requirement. But files in dashboards should be fine here. 
2. if KubeEdge authors would like to provide revised yamls, put those revised yamls in examples.

File: docs/proposals/observability-management.md (hunk 1 above)
Comment (Kanakami) [reply docs/proposals/observability-management.md]: Done

File: exporter/hardSamplesExporter.py (hunk 2 above)
Comment (Kanakami) [reply exporter/hardSamplesExporter.py]: Done

File: deployment/grafana/README.md (hunk 3 above)
Comment (Kanakami) [reply deployment/grafana/README.md]: I have deleted some original files of Grafana and update `user-guide-for-observability-management.md`. Readers can read this markdown file to learn how to install Grafana with our sedna's dashboards and specific `values.yaml`.

**Pull request merged**